### DISCUSSION TITLE
No CQ or AA visible in App

### DISCUSSION THREAD

**Discussion by donky-bison** [OfficeDev/TACO#22] [Q&A]
Hi 
When I start the app i get a screen like this - see grafik.  The AA and CQ Files in SP are populated and even the user management file is filled up with data

![image](https://user-images.githubusercontent.com/14820231/186420166-b362931d-bb5c-4f69-b501-771de80da6a1.png)

Checking the data source shows me a strange url - see  grafik 2 
![image](https://user-images.githubusercontent.com/14820231/186421324-cda923c0-1d6b-4fcd-9358-71313009c77d.png)

**Comment by JohanVeldhuis** [comment]
@donky-bison So based on this you need to update the SharePoint connection it looks like it is still referring to our test tenant which we used. Please make sure all the SharePoint connections are pointing to the site you have created to have the SPO lists hosted. 
Easiest way to do this is by opening the app in Edit mode and follow the steps in the doc below:

[Add manage connections](https://docs.microsoft.com/en-us/power-apps/maker/canvas-apps/add-manage-connections)

Based on this it looks like the SharePoint connections were not updated which is kind of weird since you mentioned the workflow has added the auto attendants and call queues to the SPO lists

**Comment by donky-bison** [comment]
No I dont' see any item on the start screen - Im starting the app with my personal account, the one I have installed the whole app

**Reply by JohanVeldhuis** [reply]
As mentioned in my previous answer it looks like the app didn't update it's data connections which results in the app connecting to the wrong SPO site. You could try to remove the connections in the edit mode of the app and recreate them and point them to the correct url. To do this perform the following steps:

- go to make.powerapps.com
- select the **solutions** item from the left menu
- find the TACO solution
- click on the **3 dots** and select **Edit**
- find the TACO app (Teams AA CQ Orchestrator (TACO)
- click on the **3 dots** and select **Edit**
- in the left menu select **data** (database icon)
- remove one of the entries and readd it by clicking **Add data**
- select **advanced** and select the variable VAR- SPO Site AutoAttendant ....
- select the list you just have removed

Johan

**Comment by JohanVeldhuis** [comment]
@donky-bison based on working with one of my colleagues we confirmed that should resolve the issue. Based on this we updated the Wiki to include these steps. [Wiki](https://github.com/OfficeDev/TACO/wiki/2.-Deployment#step-6-deploy-the-power-app-and-flows)

Let me know if this also fixes your issue

**Reply by donky-bison** [reply]
After rebuilding all the shareppoint connections there is some data showing in the app. It shows only the user management data, but no AA or CQ Data. 

![image](https://user-images.githubusercontent.com/14820231/186583511-daa34a55-367a-438b-b929-4c8e5e9bd00d.png)

I assume the filters for the call queue screen and all the other screens are wrong
![image](https://user-images.githubusercontent.com/14820231/186583653-2e368740-0f0f-49a8-94ac-f0dbb4db6c05.png)

Can you check that?

**Reply by JohanVeldhuis** [reply]
@donky-bison filters are correct we checked this in multiple environments. Did you recreated all the data connections or only the call queue one?

**Comment by donky-bison** [comment]
All of them

Von: Johan Veldhuis ***@***.***>
Gesendet: Donnerstag, 25. August 2022 08:42
An: OfficeDev/TACO ***@***.***>
Cc: Egli Urs ***@***.***>; Mention ***@***.***>
Betreff: Re: [OfficeDev/TACO] No CQ or AA visible in App (Discussion #22)


@donky-bison<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdonky-bison&data=05%7C01%7Curs.egli%40bithawk.ch%7Cb6114bc498a946e5523308da8664f3c4%7Cb58f94a366ad4b34bd9e7d8af8addb7d%7C0%7C0%7C637970065402198745%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=bYPEQ9RZbXvxv6cqBPUb2r0OqJ4Ibc%2B3RIAMVU01YWE%3D&reserved=0> filters are correct we checked this in multiple environments. Did you recreated all the data connections or only the call queue one?

-

**Reply by JohanVeldhuis** [reply]
OK how did you start the app does it show the user who has permissions in the top right corner of the app? Reason I am asking is the different between the gallery filters is that in the AA and CQ section we determine the user based on a function in PowerApps while in the gallery for the permissions we are using the O365Users filter. We validated this in multiple environments and it should work unless there is something different in your environment.

What you can do for troubleshooting is the following:

- open the app in the Editor
- add a text box
- change the value of attribute text to **locUserFullName**
- save and publish the app
- open it with the user account and validate that the value of this text field is the same as the one in the top right of the app

Johan

**Reply by donky-bison** [reply]
I found the mistake - In the user Management I have to define the exact CQ-Name or AA-Name under the column "Titel" and assign it to a user

**Reply by JohanVeldhuis** [reply]
Yeah it needs to be 100% the same else it won't find it. Good to hear you found the solution.
Just a quick check if you can remember when importing the app if it asked you to provide the SharePoint URL and some other variables.
We had one other report that those weren't asked for during the import process which might clarify the issues you reported in this thread.

**Reply by JohanVeldhuis** [reply]
@donky-bison be aware that doing this step might have broken the workflows since the connection id is different. If this is the case probably an easier way to fix all the issues is remove the solution and reimport it again, else you will need to fix all workflows manually

**Comment by donky-bison** [comment]
Hi

I think I had the same experience as described here: Questions on issues during Deployment * Discussion #8 * OfficeDev/TACO (github.com)<https://github.com/OfficeDev/TACO/discussions/8>

Best regards - Urs Egli

Von: Johan Veldhuis ***@***.***>
Gesendet: Donnerstag, 25. August 2022 09:09
An: OfficeDev/TACO ***@***.***>
Cc: Egli Urs ***@***.***>; Mention ***@***.***>
Betreff: Re: [OfficeDev/TACO] No CQ or AA visible in App (Discussion #22)


Yeah it needs to be 100% the same else it won't find it. Good to hear you found the solution.
Just a quick check if you can remember when importing the app if it asked you to provide the SharePoint URL and some other variables.
We had one other report that those weren't asked for during the import process which might clarify the issues you reported in this thread.

-

**Comment by JohanVeldhuis** [comment]
@donky-bison did you manage to fix the issue?

**Comment by donky-bison** [comment]
Sorry, I'm very busy at the moment

Von: Johan Veldhuis ***@***.***>
Gesendet: Montag, 31. Oktober 2022 08:36
An: OfficeDev/TACO ***@***.***>
Cc: Egli Urs ***@***.***>; Mention ***@***.***>
Betreff: Re: [OfficeDev/TACO] No CQ or AA visible in App (Discussion #22)


@donky-bison<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdonky-bison&data=05%7C01%7Curs.egli%40bithawk.ch%7C63727de3715e43e7173608dabb128c28%7Cb58f94a366ad4b34bd9e7d8af8addb7d%7C0%7C0%7C638027985600452932%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=wQsLnnNUldZZrF6%2F5mVUDBO5YOwBTE22l1L64rznfvA%3D&reserved=0> did you manage to fix the issue?

-
